src/ui/tabs/rooms_tab.py

```
@@ -14,9 +14,21 @@
     get_room_selection_style, get_room_group_style, get_line_edit_style,
     get_button_style
 )
-from src.core.utils import resource_path, _clear_layout # For icons and layout clearing
+from src.core.utils import resource_path # For icons
 from src.ui.custom_widgets import CustomLineEdit, AutoScrollArea, CustomSpinBox, CustomNavButton
 
+# Local _clear_layout function to avoid circular dependency with utils
+def _clear_layout(layout):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)